doc/conf.py

```diff
@@ -81,13 +81,23 @@
     "nbsphinx",
     "sphinx_autosummary_accessors",
     "scanpydoc.rtd_github_links",
```

Comment:
the edit links don't work anymore (they point at the generated files). I think this is because we switched themes and `scanpydoc.rtd_github_links` only works with `sphinx_rtd_theme` (not sure, though).

Reply:
After some digging on `scanpydoc` ext, it's my understanding that the approach used by `scanpydoc.rtd_github_links` is intertwined with the `sphinx_rtd_theme`. I was unable to get it to work with the new theme. To address this issue, I disabled the `edit on GitHub` button on the generated docstring pages, and used the `sphinx.ext.linkcode` extension to link the source code: 


<img width="712" alt="Screen Shot 2021-03-06 at 7 17 00 PM" src="https://user-images.githubusercontent.com/13301940/110226988-d3f00b00-7eb0-11eb-9fd1-23969930cc8a.png">